content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,247 @@
+| First Name / Last Name | Enter the first and last name of the person you want listed as the domain's owner. In most cases, this person is you.                                                                                                                                              |
+| Email                  | Make sure this is accurate. You'll be sent emails verifying your information that you'll be required to approve. If you enter your email address incorrectly, this email will not be delivered, and your domaiain could be suspended until it's fixed. |
+| Organization Name      | If the domain is to be registered for a company, you'll enter that here. Let's leave this blank for now.                                                                                                                                                                   |
+| Job Title              | As above, if the domain were being registered for a company, you'll enter your job title. Let's leave this blank, too.                                                                                                                                                       |
+| Phone                  | Enter your complete phone number, starting with the country code.                                                                                                                                                                                                            |
+| Fax                    | This is usually relevant for company registrations — we'll leave this blank as well.                                                                                                                                                                                           |
+| Address                | Enter your complete physical or postal address.                                                                                                                                                                                                                             |
+
+### Other options when getting started with a new account
+
+#### Adding a domain
+
+If you were purchasing a new domain, or transferring a domain in from another provider, like GoDaddy, you'd choose this. You're not transferring a domain in from ClickFunnels, because they use DNSimple as their registrar already. You're moving your domain from ClickFunnels's DNSimple account into your DNSimple account.
+
+#### Requesting an SSL certificate
+
+If you need to purchase a certificate right away, choose this option. ClickFunnels includes a certificate with their services to secure your site. We talk about this more in ["What are certificates?"](#what-are-certificates-?).
+
+#### Exploring the API
+
+The API is a great resource for those with technical backgrounds to take advantage of automation. You may want to look into it and explore more. You'll find several links to articles explaining the API at the end of this article.
```

Comment:
This mentions links to the API, but this is the only time the API is mentioned.

Reply:
Thanks for reminding me! We will add in a dump of "helpful links" at the end @deandre let's discuss Wednesday

Reply:
Is there a preference of a section at the bottom over in-line links? This article is already long enough and I think it's more helpful for relevant links to appear in context. Happy to sprinkle more in, doing so now, but I don't think there should be a separate section necessarily...feels potentially wild west to me.

Reply:
Let's move forward as is and see if we need it once it's out there